package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1800,32 +1889,17 @@ def asphericity(group, wrap=False, unwrap=None, compound='group'):
         .. versionchanged:: 2.1.0
            Renamed `pbc` kwarg to `wrap`. `pbc` is still accepted but
            is deprecated and will be removed in version 3.0.
+        .. versionchanged:: 2.5.0
+           Added use of gyration_moments for per residue quantities
```

Comment:
Is this important for the user to know _how_ the quantities are computed? I'd assume the more important message is what you have in the CHANGELOG: since 2.5.0 you can calculate asphericity for any `compound`.

Reply:
That's what I was going for with "for pre residue quantities" but hopeful the change is more clear.